tests/countries/test_belarus.py

```diff
@@ -25,37 +26,181 @@ def test_country_aliases(self):
+            ("2018-12-25", "Нараджэнне Хрыстова (каталіцкае Раство)"),
+            ("2018-12-31", "Выходны (перанесены з 29.12.2018)"),
         )
 
-    def test_new_year(self):
-        self.assertHoliday(
-            "2019-01-01",
-            "2020-01-01",
-            "2020-01-02",
-            "2021-01-01",
-            "2021-01-02",
+    def test_constitution_day(self):
+        name = "Дзень Канстытуцыі"
+
+        workdays = Belarus(categories=WORKDAY, years=range(1991, 2050))
+        self.assertHolidayName(name, workdays, (f"{year}-03-15" for year in range(1999, 2050)))
+        self.assertNoHolidayName(name, workdays, (f"{year}-03-15" for year in range(1991, 1995)))
+        self.assertHolidayName(name, (f"{year}-03-15" for year in range(1995, 1999)))
+        self.assertNoHolidayName(
+            name,
+            (f"{year}-03-15" for year in range(1991, 1995)),
+            (f"{year}-03-15" for year in range(1999, 2050)),
+        )
+
+    def test_day_of_unity_of_the_peoples_of_belarus_and_russia(self):
+        name = "Дзень яднання народаў Беларусі і Расіі"
+
+        workdays = Belarus(categories=WORKDAY, years=range(1991, 2050))
+        self.assertHolidayName(name, workdays, (f"{year}-04-02" for year in range(1996, 2050)))
+        self.assertNoHolidayName(name, workdays, (f"{year}-04-02" for year in range(1991, 1996)))
+        self.assertNoHolidayName(name)
+
+    def test_national_symbol_day(self):
+        name = "Дзень Дзяржаўнага сцяга, Дзяржаўнага герба і Дзяржаўнага гімна Рэспублікі Беларусь"
+
+        dt = (
+            "2012-05-13",
+            "2013-05-12",
+            "2014-05-11",
+            "2015-05-10",
+            "2016-05-08",
+            "2017-05-14",
+            "2018-05-13",
+            "2019-05-12",
+            "2020-05-10",
+            "2021-05-09",
+            "2022-05-08",
+            "2023-05-14",
+            "2024-05-12",
+            "2025-05-11",
+            "2026-05-10",
+            "2027-05-09",
+            "2028-05-14",
+            "2029-05-13",
+            "2030-05-12",
         )
+        workdays = Belarus(categories=WORKDAY, years=range(1991, 2050))
+        self.assertHolidayName(name, workdays, dt)
+        self.assertNoHolidayName(name, workdays, range(1991, 1998))
+        self.assertNoHolidayName(name)
+
+    def test_day_of_the_republic(self):
+        name = "Дзень Незалежнасці Рэспублікі Беларусь (Дзень Рэспублікі)"
+
+        self.assertHolidayName(name, (f"{year}-07-27" for year in range(1991, 1997)))
+        self.assertHolidayName(name, (f"{year}-07-03" for year in range(1997, 2050)))
+        self.assertNoHolidayName(name, (f"{year}-07-27" for year in range(1997, 2050)))
+        self.assertNoHolidayName(name, (f"{year}-07-03" for year in range(1991, 1997)))
+
+    def test_day_of_peoples_unity(self):
+        name = "Дзень народнага адзінства"
+
+        workdays = Belarus(categories=WORKDAY, years=range(1991, 2050))
```

Comment:
I suggest to add
```python
    cls.workday_holidays = Belarus(categories=WORKDAY, years=range(1991, 2050))
```
into `setUpClass(cls)` and use it when needed (as `self.workday_holidays`).